v2/clients/http/notification.go

```diff
@@ -0,0 +1,158 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"net/url"
+	"path"
+	"strconv"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/http/utils"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/requests"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/responses"
+)
+
+type NotificationClient struct {
+	baseUrl string
+}
+
+// NewNotificationClient creates an instance of NotificationClient
+func NewNotificationClient(baseUrl string) interfaces.NotificationClient {
+	return &NotificationClient{
+		baseUrl: baseUrl,
+	}
+}
+
+// Add adds new notifications.
+func (client *NotificationClient) Add(ctx context.Context, reqs []requests.AddNotificationRequest) (res []common.BaseWithIdResponse, err errors.EdgeX) {
```

Comment:
Is `Add` what used to be `SendNotification` in V1? IMO, from a usage perspective, I prefer `SendNotification`

Reply:
> Is Add what used to be SendNotification in V1?

Yes. 
Renamed the func to make it more sensible.